diff --git a/cpp/open3d/core/linalg/BlasWrapper.h b/cpp/open3d/core/linalg/BlasWrapper.h
index a5893d39..61582eef 100644
--- a/cpp/open3d/core/linalg/BlasWrapper.h
+++ b/cpp/open3d/core/linalg/BlasWrapper.h
@@ -26,6 +26,9 @@
 
 #pragma once
 
+#if __aarch64__
+#include "/usr/include/aarch64-linux-gnu/cblas-netlib.h" 
+#endif
 #include "open3d/core/linalg/LinalgHeadersCPU.h"
 #include "open3d/core/linalg/LinalgHeadersCUDA.h"
 #include "open3d/utility/Console.h"
diff --git a/cpp/open3d/geometry/Line3D.cpp b/cpp/open3d/geometry/Line3D.cpp
index 28e2e6c6..40851253 100644
--- a/cpp/open3d/geometry/Line3D.cpp
+++ b/cpp/open3d/geometry/Line3D.cpp
@@ -27,7 +27,7 @@ Line3D::Line3D(const Eigen::Vector3d& origin,
 }
 
 void Line3D::Transform(const Eigen::Transform<double, 3, Eigen::Affine>& t) {
-    this->transform(t);
+    this->Transform(t);
 }
 
 std::pair<double, double> Line3D::SlabAABBBase(
@@ -321,7 +321,7 @@ Segment3D::Segment3D(const std::pair<Eigen::Vector3d, Eigen::Vector3d>& pair)
     : Segment3D(std::get<0>(pair), std::get<1>(pair)) {}
 
 void Segment3D::Transform(const Eigen::Transform<double, 3, Eigen::Affine>& t) {
-    this->transform(t);
+    this->Transform(t);
     end_point_ = t * end_point_;
 }
 
